docs/src/joint/api/dia/CellView/prototype/findPortNode.html

```diff
@@ -1,5 +1,5 @@
 <pre class="docs-method-signature"><code>cellView.findPortNode(portId)</code></pre>
-<p>Return the port element of this CellView that is identified by <code>portId</code> (i.e. the SVGElement within <code>this.el</code> which has <code>'port': portId</code>).</p>
+<p>Return the port element of this CellView that is identified by <code>portId</code> (i.e. an SVGElement within <code>this.el</code> which has <code>'port': portId</code> / an SVGElement referenced by <code>portRoot</code> selector).</p>
```

Comment:
"... an SVGElement referenced by **the** <code..."

not sure if you meant to have `/` in this line. If yes then fine. I think a comma would be fine too.
`.../code>, an SVGElement...`